### PR TITLE
Create color picker input

### PR DESCRIPTION
**Pull request by chiragzq**
Color picker, with manual RGB/HSV adjust

### CODE REVIEW

File: src/components/baklava-options-plugin/ColorPickerOption.vue [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+export default class ColorPickerOption extends Vue {
+  @Prop({ type: String })
+  name!: string;
+
+  @Prop()
+  // eslint-disable-next-line
+  value!: any; // it has to be any; the baklava source does this
+
+  @Prop({ type: String, default: "#000000" })
+  displayValue!: string;
```
Comment (carbotaniuman) [inline src/components/baklava-options-plugin/ColorPickerOption.vue]: Don't need the whole displayValue thing anymore, just use Baklava's `this.value` prop

File: src/components/baklava-options-plugin/ColorPickerOption.vue [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+@Component({
+  components: { "chrome-picker": Chrome },
+})
+export default class ColorPickerOption extends Vue {
+  @Prop({ type: String })
+  name!: string;
+
+  @Prop()
+  // eslint-disable-next-line
+  value!: any; // it has to be any; the baklava source does this
```
Comment (carbotaniuman) [inline src/components/baklava-options-plugin/ColorPickerOption.vue]: You can make it string, TypeScript is an unsound type system

File: src/components/baklava-options-plugin/ColorPickerOption.vue [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+  }
+
+  endEdit() {
+    this.isOpen = false;
+    this.$emit("input", this.displayValue);
+  }
+}
+</script>
+
+<style>
```
Comment (carbotaniuman) [inline src/components/baklava-options-plugin/ColorPickerOption.vue:43]: Try <style scoped>

File: src/components/node-editor/nodeFromSchema.ts [hunk 4]
```diff
@@ -15,6 +15,8 @@ function defaultValue(io: InputOutput): unknown {
       return io.params.min ?? 0;
     case "Enum":
       return (io.params.items as unknown[])[0];
+    case "Color":
+      return "#95C624";
```
Comment (carbotaniuman) [inline src/components/node-editor/nodeFromSchema.ts:19]: Any reason why this color in particular?

File: src/components/baklava-options-plugin/ColorPickerOption.vue [hunk 5]
```diff
@@ -0,0 +1,89 @@
+<template>
+  <div class="color-picker-container">
+    <div class="__label .text-truncate color-picker-label">{{ name }}</div>
+    <div
+      class="color-picker-box"
+      :style="background"
```
Comment (carbotaniuman) [inline src/components/baklava-options-plugin/ColorPickerOption.vue]: `v-bind:style="{ backgroundColor: value}"` would work just as well

**Review by carbotaniuman — CHANGES_REQUESTED** [review]

File: src/components/node-editor/nodeFromSchema.ts (hunk 4 above)
Comment (chiragzq) [reply src/components/node-editor/nodeFromSchema.ts:19]: It's the green in the opensight logo

**Review by carbotaniuman — APPROVED** [review]

**Review by 132ikl — CHANGES_REQUESTED** [review]
Can you make a nicer style to fit more with the rest of the frontend, and remove the alpha parameter? Other than that, looks good.

**Comment by chiragzq** [comment]
(and remove alpha slider)

**Review by 132ikl — APPROVED** [review]

**Pull request merged**